src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,68 @@
+import classnames from 'classnames';
+import { FilterType } from '../../types/FilterStatus';
+
+import { Props } from './Footer.props';
+
+export const Footer: React.FC<Props> = ({
+  filterType,
+  handleFilterType,
+  todos,
+  deleteCompleted,
+}) => {
+  const todosLeft = todos.filter(todo => !todo.completed).length;
+  const todosCompleted = todos.filter(todo => todo.completed).length;
```

Comment:
```suggestion
  const todosCompleted = todos.length - todosLeft;
```

Redundant filter operation (simplify complexity)